src/Microsoft.Dnx.Tooling/NuGet/ContentModel/Infrastructure/Parser.cs

```diff
@@ -110,7 +110,7 @@ internal override bool TryMatch(ContentItem item, IDictionary<string, ContentPro
                 ContentPropertyDefinition propertyDefinition;
```

Comment:
It seems like all the files under Tooling/Nuget/ContentModel don't have copyright. Is this intentional?